src/RabbitMQAttribute.cs

```diff
@@ -40,8 +39,5 @@ public sealed class RabbitMQAttribute : Attribute
 
         [ConnectionString]
         public string ConnectionStringSetting { get; set; }
-
-        [AutoResolve]
-        public string DeadLetterExchangeName { get; set; }

```

Comment:
Without this users will not be able to specify an existing queue for `DeadLetterExchangeName` ?